cumulus_etl/deid/ms-config.json

```diff
@@ -156,6 +228,43 @@
     {"path": "Encounter.serviceProvider", "method": "keep"},
     {"path": "Encounter.partOf", "method": "keep"},
 
+    {"comment": "** https://www.hl7.org/fhir/R4/immunization.html **", "path": "xxx", "method": "redact"},
+    {"path": "Immunization.status", "method": "keep"},
+    {"path": "Immunization.statusReason", "method": "keep"},
+    {"path": "Immunization.vaccineCode", "method": "keep"},
+    {"path": "Immunization.patient", "method": "keep"},
+    {"path": "Immunization.encounter", "method": "keep"},
+    {"path": "Immunization.occurrence.ofType(string)", "method": "redact", "comment": "would run philter on it, but it would just remove anything useful we could parse"},
+    {"path": "Immunization.occurrence", "method": "keep"},
+    {"path": "Immunization.recorded", "method": "keep"},
+    {"path": "Immunization.primarySource", "method": "keep"},
+    {"path": "Immunization.reportOrigin", "method": "keep"},
+    {"path": "Immunization.location", "method": "keep"},
+    {"path": "Immunization.manufacturer", "method": "keep"},
+    {"path": "Immunization.lotNumber", "method": "keep", "comment": "caution: non-PHI freeform string"},
+    {"path": "Immunization.expirationDate", "method": "keep"},
+    {"path": "Immunization.site", "method": "keep"},
+    {"path": "Immunization.route", "method": "keep"},
+    {"path": "Immunization.doseQuantity", "method": "keep"},
+    {"path": "Immunization.performer.function", "method": "keep"},
+    {"path": "Immunization.performer.actor", "method": "keep"},
+    {"path": "Immunization.reasonCode", "method": "keep"},
+    {"path": "Immunization.reasonReference", "method": "keep"},
+    {"path": "Immunization.isSubpotent", "method": "keep"},
+    {"path": "Immunization.subpotentReason", "method": "keep"},
+    {"path": "Immunization.education.publicationDate", "method": "keep", "comment": "we aren't keeping documentType or uri for fear of any PHI slipping in, so maybe these two dates aren't very useful, but better more data than less"},
```

Comment:
I'm still not 100% on whether it's safe to include `documentType`/`uri` -- they seem kinda harmless, but I also don't think they would be very valuable... :shrug: